### ISSUE TITLE
image-url() should be removed?

### ISSUE THREAD

**Issue by joelpittet** [Igosuki/compass-mixins#48]
Noticed many references to image-url(). Does that work with libsass?

**Comment by sh1989** [comment]
You're right. Libsass [doesn't provide implementations of functions](https://github.com/sass/libsass/issues/489) such as image-url, font-url, etc. They should be removed or implemented via a custom function in compass-mixins.

**Comment by strarsis** [comment]
It is possible to use JavaScript/node functions with libsass using [eyeglass](https://github.com/sass-eyeglass/eyeglass) (or [sassport](https://github.com/davidkpiano/sassport)).
For inlining SVGs, this eyeglass module can be used: [eyeglass-inline-svg](https://github.com/strarsis/eyeglass-inline-svg).

**Comment by warrickhill** [comment]
Any update to this?

**Comment by strarsis** [comment]
@warrickhill: node-sass [supports custom functions](https://github.com/sass/node-sass/pull/644).
[eyeglass](https://github.com/sass-eyeglass/eyeglass) takes it further and automatically adds these custom functions from installed npm packages.
This in turn can be used to add the functionality to node-sass/libsass (see my previous post).
